<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GammaRay::PaintBufferViewer</class>
 <widget class="QWidget" name="GammaRay::PaintBufferViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>378</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTreeView" name="commandView">
      <property name="rootIsDecorated">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="replayContainer">
       <item>
        <widget class="GammaRay::RemoteViewWidget" name="replayWidget" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GammaRay::RemoteViewWidget</class>
   <extends>QWidget</extends>
   <header>remoteviewwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
